src/custom/hooks/useFilterEmptyQueryParams.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+import qs from 'qs'
+
+import useParsedQueryString from '@src/hooks/useParsedQueryString'
```

Comment:
nit pick: remove the `@src` prefix

Reply:
I did it.
What is the reason for this? It is only used for everything that is not custom?

Reply:
`@src` is meant to be used only when we really want to use the original version. That applies most of the time to original Uniswap source files and components modding them

All other cases should use the regular path.
In case we ever need to patch it, we then use our modding pattern creating a module with the same path inside the `/custom` path.

For example, in the connectors mod, we want to re-export all the original exports https://github.com/gnosis/cowswap/blob/f8c69bee893248404f37d81d0248076b155d93de/src/custom/connectors/index.ts#L11